docs/codeflow/codeflow-faq.md

```diff
@@ -1,5 +1,9 @@
 ---
 title: Codeflow FAQ
+head:
+  - - meta
```

Comment:
I find this syntax a bit arcane. Would it be better to use something like this?

```yml
head:
  - ['meta', {property: 'og:image', content: 'https://developer.stackblitz.com/img/og/codeflow-faq.png'}]
```

Reply:
I'm fine with that